examples/Demo/Shared/Components/Cookies/ManageCookies.razor

```diff
@@ -0,0 +1,70 @@
+            <FluentRadioGroup Class="choices" Name="AcceptAnalytics" @bind-Value="@Content.AcceptAnalytics">
+                <FluentRadio Value="true">Accept</FluentRadio>
+                <FluentRadio Value="false">Reject</FluentRadio>
+            </FluentRadioGroup>
+        </dt>
+
+        <dt aria-label="Social Media">
+            <h4>Social Media</h4>
+            <p class="explainer">We and third parties use social media cookies to show you ads and content based on your social media profiles and activity on our websites. They’re used to connect your activity on our websites to your social media profiles so the ads and content you see on our websites and on social media will better reflect your interests.</p>
+            <FluentRadioGroup Class="choices" Name="AcceptAnalytics" @bind-Value="@Content.AcceptSocialMedia">
+                <FluentRadio Value="true">Accept</FluentRadio>
+                <FluentRadio Value="false">Reject</FluentRadio>
+            </FluentRadioGroup>
+        </dt>
+
+        <dt aria-label="Advertising">
+            <h4>Advertising</h4>
+            <p class="explainer">We and third parties use advertising cookies to show you new ads by recording which ads you've already seen. They're also used to track which ads you click on or purchases you make after clicking on an ad for payment purposes, and to show you ads that are more relevant to you. For example, they're used to detect when you click on an ad and show you ads based on your social media interests and website browsing history.</p>
+            <FluentRadioGroup Class="choices" Name="AcceptAnalytics" @bind-Value="@Content.AcceptAdvertising">
+                <FluentRadio Value="true">Accept</FluentRadio>
+                <FluentRadio Value="false">Reject</FluentRadio>
+            </FluentRadioGroup>
+        </dt>
+    </dl>
+</FluentDialogBody>
+<FluentDialogFooter>
+    <FluentButton Style="min-width: 150px;" Disabled="@_buttonsDisabled" @onclick="HandleSaveAsync" Appearance="Appearance.Accent">Save changes</FluentButton>
+    <FluentButton Style="min-width: 150px;" Disabled="@_buttonsDisabled" @onclick="HandleCancelAsync">Cancel</FluentButton>
+</FluentDialogFooter>
+
+@code {
+    private bool _buttonsDisabled => Content.AcceptAnalytics is null && Content.AcceptSocialMedia is null && Content.AcceptAdvertising is null;
+
+    [Parameter]
+    public CookieState Content { get; set; } = default!;
+
+    [CascadingParameter]
+    public FluentDialog? Dialog { get; set; }
+
+
+    private async Task HandleSaveAsync()
+    {
+        await Dialog!.CloseAsync(Content);
```

Comment:
I presume that the Dialog will always defined?
So, you can set `public FluentDialog? Dialog { get; set; } = default!;` and remove the `!` for `Dialog!...`